pyomo/contrib/gdpopt/util.py

```diff
@@ -350,16 +353,20 @@ def build_ordered_component_lists(model, solve_data):
             model.component_data_objects(
                 ctype=Constraint, active=True,
                 descend_into=(Block, Disjunct))))
+    if hasattr(solve_data,'mip_constraint_polynomial_degree'):
+        mip_constraint_polynomial_degree = solve_data.mip_constraint_polynomial_degree
+    else:
+        mip_constraint_polynomial_degree=(0, 1)
```

Comment:
It would be slightly more efficient if this was a set and not a tuple.

Reply:
Done